overview.md

```diff
@@ -0,0 +1,158 @@
+# SourceCred
+
+SourceCred is anchored by a simple goal: we want open-source contributors to
```

Comment:
Is it that you "want open-source contributors to
earn a living from their work", or do you want the, to earn a living AND share in the rewards of people creating value based on their work

Reply:
The latter is a better description of our goal. Updated.